app/assets/js/script.js

```diff
@@ -3,7 +3,17 @@
   'use strict';
   var $win = $(window);
   var $doc = $(document);
-
+  
+  $(document).ready(function () {
+    $(window).scroll(function () {
+      if ($(window).scrollTop() > 148) {
+        $('.context-nav').addClass('navbar-fixed');
+      }
+      if ($(window).scrollTop() < 149) {
+        $('.context-nav').removeClass('navbar-fixed');
+      }
```

Comment:
`$('.context-nav').toggleClass('navbar-fixed', $win.scrollTop() > 148);` accomplishes the same thing as lines 9-14